_data/people.yml

```diff
@@ -334,3 +334,14 @@
     - climate
     - software
     - GIS
+  
+ - first_name: Yu
+  last_name: Cheng
```

Comment:
Please fix the formatting of the .yml entry. Everything needs to be aligned properly or else it will not build.